provision/PROVISION.md

```diff
@@ -0,0 +1,53 @@
+## Provision
+
+>Note: This provisioning model only supports Keycloak Legacy distribution, we are working on a version to support Quarkus based distribution.
+
+#### Keycloak
+A light weight Keycloak server setup with, PostgreSQL DB as its external store using docker-compose.
+
+ - The image used for Keycloak server is the latest:legacy container from [quay.io](https://quay.io/repository/keycloak/keycloak?tab=tags&tag=legacy)
+ - To control the cpu quota for the containers, we use cpusets for both containers (keycloak and postgres), that can be changed from the [keycloak-legacy-postgres.yml](keycloak/keycloak-legacy-postgres.yml)
+ 
+##### Pre-requisite
+The keycloak-legacy-postgres.yml file should work with the below docker and docker-compose versions
+
+Docker CE : 20.10.12
+Docker Compose: 1.29.2
+
+#### Bringing up the containers
+
+To run in a detached mode,
+
+```shell
+docker-compose -f keycloak-postgres.yml up -d
+```
+
+- The Keycloak application should be available at http://localhost:8080/
+- The PostgreSQL db should be available at localhost:5432
+
+To view individual container logs,
+
+```shell
+docker-compose -f keycloak-legacy-postgres.yml logs postgres
+```
+
+```shell
+docker-compose -f keycloak-legacy-postgres.yml logs keycloak
+```
+
+_Note: It would be a good idea, to run the below command with some frequency, to pull the latest versions of the images on your localhost._
+
+```shell
+docker-compose pull
+```
+
+
+##### Deploying the dataset module custom provider to this containerized Keycloak distribution
+
+Assuming you have already read the [Dataset Module Readme](../README.md)
+
+```shell
+docker cp ../dataset/target/keycloak-benchmark-dataset-*.jar $keycloak:/opt/jboss/keycloak/standalone/deployments/
+```
+
+This should be a hot deploy, so restart of the container is probably not necessary.
```

Comment:
How can I test that this worked: Is there a log on the console? Is there a CURL endpoint I could call to verify that the installation occurred?

Reply:
makes sense, updated the console log example.